docs/conf/prague/2020/news/prg-covid-19-update-01.rst

```diff
@@ -0,0 +1,40 @@
+:template: {{year}}/generic.html
+
+.. post:: March 16, 2020
+   :tags: prague-2020, covid-19
+
+Prague Conference - COVID-19 Update 01
+======================================
+
+The most recent COVID-19 update is:
+
+.. postlist:: 1
+  :date: %A, %B %d, %Y
+  :format: {title} - {date}
+  :list-style: none
+  :tags: covid-19
```

Comment:
I don't think we need this yet, since we only have 1. We added this after we had more than 1.